test/docfx.Test/data/validation/schema.json

```diff
@@ -5234,6 +5236,11 @@
         "severity": "suggestion",
         "code": "duplicate-description-within-docset",
         "additionalMessage": "Description must be unique within a docset."
+      },
+      "description-bad-length": {
+        "severity": "suggestion",
+        "code": "description-bad-length",
+        "additionalMessage": "Description length of characters is out of range. The description should be between MinLength and MaxLength characters, including spaces."
```

Comment:
would it be better that we include the actual min and max number in error message?

Reply:
That is already included in the first part of the message. This is `additionalMessage` that is appended to the general message.

Reply:
the message can be found in yml test case.